src/scripts.js

```diff
@@ -3,31 +3,58 @@
 let greeting = document.querySelector('.greeting');
 let stepGoalComparison = document.querySelector('.you-vs-others-data');
 
+
+let userCardID = document.querySelector('.user-card-id');
+let userCardName= document.querySelector('.user-card-name');
+let userCardAddress = document.querySelector('.user-card-address');
+let userCardEmail = document.querySelector('.user-card-email');
+let userCardStrideLength = document.querySelector('.user-card-stride-length');
+let userCardDailyStepGoal = document.querySelector('.user-card-daily-step-goal');
+let userCardFriends = document.querySelector('.user-card-friends');
+
+
 let userRepository = new UserRepository(userData);
+let user = createUser();
```

Comment:
I wonder if there is a better way to get this done (making our user part of the global scope).  
Let's keep this in mind moving forward, maybe as a refactor.

Reply:
Let's start considering this now. It may make everything easier moving forward.